Orange/widgets/unsupervised/owdistancetransformation.py

```diff
@@ -69,5 +73,17 @@ def commit(self):
             distances = inv(distances)
         self.send("Distances", distances)
 
+    def send_report(self):
+        norm, normopt = self.normalization_method, self.normalization_options
+        inv, invopt = self.inversion_method, self.inversion_options
+        parts = []
+        if inv:
+            parts.append('inversion ({})'.format(invopt[inv][0]))
+        if norm:
+            parts.append('normalization ({})'.format(normopt[norm][0]))
+        self.report_items('Model parameters',
+                  (('Transformation', ', '.join(parts).capitalize() or 'None'),))
```

Comment:
Oh, and could you please indent this hanging line [according to PEP8](https://www.python.org/dev/peps/pep-0008/#indentation)?

Perhaps while at it, since Transformation is the only argument, you can also make it a dict:
```py
{'Transformation': ', '.join(parts).capitalize() or 'None'}
```
It looks a bit more clear like this, I agree. :flushed: